client/src/redux/user/action.js

```diff
@@ -0,0 +1,8 @@
+export const setCurrentUser=user=>({
+    type:'SET_CURRENT_USER',
```

Comment:
add a seperate file for all the types. user/types.js . Just a simple convention

Reply:
ok sure .